docs/28.API-Tests.md

```
@@ -0,0 +1,605 @@
+* [tests/test_api.py L232-L238](https://github.com/ArWikiCats/ArWikiCatsWeb/blob/88f42d13/tests/test_api.py#L232-L238)  - `TestListEndpoint`
+* [tests/test_user_agent.py L14-L19](https://github.com/ArWikiCats/ArWikiCatsWeb/blob/88f42d13/tests/test_user_agent.py#L14-L19)  - `TestUserAgentHeader`
+
+**Sources:** [tests/test_api.py L82-L88](https://github.com/ArWikiCats/ArWikiCatsWeb/blob/88f42d13/tests/test_api.py#L82-L88)
+
+ [tests/test_user_agent.py L14-L19](https://github.com/ArWikiCats/ArWikiCatsWeb/blob/88f42d13/tests/test_user_agent.py#L14-L19)
+
+### Response Validation Patterns
+
+**JSON Response Parsing:**
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)